execution/src/transaction.rs

```diff
@@ -234,6 +234,27 @@ impl NormalizedExtendedTxEnvelope {
     }
 }
 
+type MoveChanges = ChangeSet;
+type EvmChanges = moved_evm_ext::state::StorageTriesChanges;
+
+#[derive(Debug, Clone)]
+pub struct Changes {
+    pub r#move: MoveChanges,
```

Comment:
I remember there was some discussion before, but how about just calling it `moved`? So that we don't have to make it a raw ident. Though it's a nit, of course.

Reply:
I think the best name is `move_vm` as it contains the name of the embedded environment where the changes were generated.